package.json

```diff
@@ -5,9 +5,13 @@
   "dependencies": {
     "@ant-design/icons": "^4.2.2",
     "@craco/craco": "^5.6.4",
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
     "@material-ui/core": "^4.11.4",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.58",
+    "@mui/icons-material": "^5.11.0",
```

Comment:
Don't use v5 mui.
The app works on v4 material ui so us e its icons

use `@material-ui/icons` : 4.11.2

Reply:
Changed the version from 5 to 4.